worlds/noita/__init__.py

```diff
@@ -0,0 +1,111 @@
+import string
+from typing import Dict, List
+from .Items import NoitaItem, item_table, item_pool_weights
+from .Locations import NoitaLocation, item_pickups
+
+from BaseClasses import Region, RegionType, Entrance, Item, ItemClassification, MultiWorld, Tutorial
+from .Options import noita_options
+from worlds.AutoWorld import World, WebWorld
+
+client_version = 1
+
+
+class NoitaWeb(WebWorld):
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Noita integration for Archipelago multiworld games.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["DaftBrit"]
+    )]
+    theme = "partyTime"
+
+
+class NoitaWorld(World):
+    """
+    Noita is a magical action roguelite set in a world where every pixel is physically simulated. Fight, explore, melt,
+    burn, freeze and evaporate your way through the procedurally generated world using spells you've created yourself.
+    """
+    game: str = "Noita"
+    option_definitions = noita_options
+    topology_present = False
+
+    item_name_to_id = item_table
+    location_name_to_id = item_pickups
+
+    data_version = 5
+    forced_auto_forfeit = True
```

Comment:
Preferable to not do this. If you don't have any sort of user text input for sending messages they can still forfeit through the text client, which you can recommend in the guide.